### ISSUE TITLE
Possible error in curvature computation

### ISSUE THREAD

**Issue by bartmate** [udacity/sdc-issue-reports#394]
Hi,

in the sublesson 34. Measuring Curvature:
When we measure the curvature in the real-world space, we cannot use the polynom what we calculated in the pixel space, as it is written in the description at the bottom of the lesson. So we have to convert the lane line point from the pixel space to real-world space (by multiplying the x and y coordinates by xm_per_pix and ym_per_pix, respectively). Then we can calculate the curvature in the real world. 

My problem here is why we use y_eval in the calculation. y_eval is in pixel space, typically it is around ~720. I think it should also be converted to real world space, i.e. we should multiply it by ym_per_pix. So the correct formula:

((1 + (2*left_fit_cr[0]*(y_eval*ym_per_pix) + left_fit_cr[1])**2)**1.5) \
                             /np.absolute(2*left_fit_cr[0])

This will result a much smaller value.

Regards,
  Mate

**Comment by awbrown90** [comment]
@bartmate, agreed. I did this as well on my project and got much better results for curvature when I did.

**Comment by ryan-keenan** [comment]
Thanks for catching this! there was an error in the code there and it has been corrected now ;).
